prime-router/src/main/kotlin/Translator.kt

```diff
@@ -46,8 +38,9 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
                 (limitReceiversTo.isEmpty() || limitReceiversTo.contains(receiver.fullName))
         }.mapNotNull { receiver ->
             try {
-                val mappedReport = translateByReceiver(input, receiver, defaultValues)
-                if (mappedReport.itemCount == 0) return@mapNotNull null
+                val jurisFilteredReport = filterByJurisdiction(input, receiver)
+                if (jurisFilteredReport.isEmpty()) return@mapNotNull null
```

Comment:
These steps were split apart (filterByJursdiction, then filter by quality + translate) to allow a report that was empty due to quality filtering to be returned.. while keeping reports filtered based on jurisdiction out.